src/modules/gcloud.rs

```diff
@@ -88,6 +88,12 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     let gcloud_context = GcloudContext::new(&config_name, &config_path);
     let account: Lazy<Option<Account<'_>>, _> = Lazy::new(|| gcloud_context.get_account());
 
+    // Disable the Module if gcloud's active config is set to "NONE".
+    // This is a valid gcloud configuration.
+    if gcloud_context.config_name == "NONE" {
+        return None;
```

Comment:
This line isn't covered by the tests, which likely means the test doesn't work as expected.

Reply:
So I'm looking at this now. Originally I used `region_not_set_with_display_region` as a reference.
https://github.com/starship/starship/blob/b9746100e2ee3746ea418e26ebc50494710a9d03/src/modules/gcloud.rs#L396-L410

Looking into `ModuleRenderer`, isn't `collect()` supposed to return `None` for tests like these? 
https://github.com/starship/starship/blob/b9746100e2ee3746ea418e26ebc50494710a9d03/src/test/mod.rs#L156-L159

Maybe I'm missing something. I can look at how `module(context)` is utilized, but I want to make sure I'm on the right track first.

Reply:
You'll need to base the test on something that would usually return `Some(...)`, but returns `None` instead because the active config is `NONE`.